apps/prairielearn/src/middlewares/content-security-policy.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+
+const router = Router();
+
+router.all('/*', function (req, res, next) {
+  // Prevent PrairieLearn from being rendered in an iframe.
+  res.header('Content-Security-Policy', "frame-ancestors: 'none';");
```

Comment:
That worked in my testing, once LTI was configured to load in a new window. The looser "frame-ancestors: 'self'" worked too.

Is that the only CSP you want to add, or do we want to enforce other things too?

Reply:
I'm laser-focused on this for now. There's probably more we can do with CSP, but I'll save that for the future.

Reply:
Related, this might be of interest. https://helmetjs.github.io/faq/you-might-not-need-helmet/ Note that it contains CSP and some of the older X-Header options, like X-Frame-Options